### PR TITLE
build sea for windows macos and linux

### PR DESCRIPTION
**Pull request by Brocco**

### CODE REVIEW

File: .github/workflows/build-seas.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+name: Runtime Tests
+on:
+  workflow_call:
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (aavorthmann) [inline .github/workflows/build-seas.yml:5]: should we name this token for what it is so we can find it ?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -47,3 +53,65 @@ jobs:
...
+        with:
+          upload_url: ${{ steps.create-release.outputs.upload_url }}
+          asset_path: ./dist/apps/cli/hdcli-macos.zip
+          asset_name: hdcli-macos.zip
+          asset_content_type: application/zip
+
+      - name: upload windows zip asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (aavorthmann) [inline .github/workflows/release.yml:112]: same question regarding naming

**Review by aavorthmann — APPROVED** [review]
minor question

File: .github/workflows/build-seas.yml (hunk 1 above)
Comment (jeremymwells) [reply .github/workflows/build-seas.yml:5]: ~It's an org token, for making requests to github. It's used in many places/repos~
This being a herodevs repo, I don't know anything about it.

**Review by jeremymwells — APPROVED** [review]

**Pull request merged**